src/adt/image.rs

```diff
@@ -1,23 +1,21 @@
 use elementtree::Element;
 use serde::{Deserialize, Serialize};
 
-use super::foundation::{Size, SizeOptionT, SizeOption, Position, PositionOptionT, PositionOption};
+use super::foundation::{Position, PositionOption, PositionOptionT, Size, SizeOption, SizeOptionT};
 use super::svg::SvgTangibleObject;
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Image {
     path: String,
+    round: Option<usize>,
```

Comment:
`round` sounds like a common property to elements. When we implement this for a second element we need to figure out how to generalize the implementation better (in my unfortunate experience, people tend to forget this and just copy-paste code later, but I trust this project better :) ).

Reply:
We also need to figure out why test coverage is complaining here, weird.